ci/run_ci.sh

```diff
@@ -146,10 +146,11 @@ integration_tests() {
 
 jdk17_plus_tests() {
   java -version
+  export JDK_JAVA_OPTIONS="--add-opens=java.base/java.nio=org.apache.arrow.memory.core,ALL-UNNAMED"
   echo "Executing fury java tests"
   cd "$ROOT/java"
   set +e
-  mvn -T10 --batch-mode --no-transfer-progress test install -pl '!fury-format,!fury-testsuite'
+  mvn -T10 --batch-mode --no-transfer-progress install
```

Comment:
Is there a reason fury-format is excluded here? I think it means integration tests can run with the wrong fury-format jar - one from a snapshot repo, instead of the one from the PR build

Reply:
> Is there a reason fury-format is excluded here? I think it means integration tests can run with the wrong fury-format jar - one from a snapshot repo, instead of the one from the PR build

It should not be skipped, we skipped before just because running arrow on JDK17+ in earlier version is tricky